### DISCUSSION TITLE
Nominatim 4.0 compatibility

### DISCUSSION THREAD

**Discussion by otbutz** [komoot/photon#613] [Q&A] [answered]
Looks like the new Nominatim 4.0 release changed quite a few things. Is the current version of Photon still compatible?

**Comment by lonvia** [comment, marked as answer]
Yes. Nothing has changed in the tables that Photon queries. However, the update mechanism has changed, so it is no longer possible to run updates against a Nominatim 4.x instance, see also #638.
